tests/test_interpolatedimage.py

```diff
@@ -1166,11 +1166,23 @@ def test_ne():
     perturb_image.array[64, 64] *= 1000
     obj2 = galsim.InterpolatedImage(perturb_image, calculate_maxk=False, calculate_stepk=False)
 
+    # These tests won't always work if astropy < 1.0.6 has been imported, so look for that.
+    try:
+        import sys
+        if 'astropy' in sys.modules:
+            import astropy  # Just b/c someone imported it, doesn't mean we can see it yet.
+            from distutils.version import LooseVersion
+            if LooseVersion(astropy.__version__) < LooseVersion('1.0.6'):
+                return
+    except:
+        pass
```

Comment:
Whoops.  I think I was trying to catch an ImportError when importing astropy.  If it's already in sys.modules though, this is probably unnecessary, so I just removed the try/except block.